scripts/menu.sh

```diff
@@ -442,9 +433,9 @@ function configure_gke() {
   local all_quotas=$(gcloud compute project-info describe \
                      --flatten "quotas[]" \
                      --format "value(quotas.metric, quotas.limit)")
-  # Check if firewalls is 200, which is standard for an upgraded project
+  # Check if firewalls is 100, which is standard for an upgraded project
   local firewalls=$(echo "${all_quotas}" | grep FIREWALLS | awk '{print int($2)}')
-  if [ $firewalls -lt 200 ]; then
+  if [ $firewalls -lt 100 ]; then
```

Comment:
Can you explain this value to me? I'm not familiar with the firewall resource.

Reply:
It was a means to check on the status of a google account upgrade once upon a time.

Reply:
an "upgraded" google cloud account has a default firewall quota of 100, so if it is less than that, they must be using a non-upgraded account.

Reply:
OK, this sounds good. I'm not familiar with clusters being upgraded or not, though, so I just want to confirm that this is something we still want.

Reply:
It's not the cluster that is upgraded, its the Google Cloud account itself.

In the Getting Started instructions, we have people "Upgrade" their account so that they are able to request GPU quotas, etc. This is just a check to make sure they have completed that step, and if not, instructs them to do so. I think that this is something we want to keep.

Reply:
Got it.